sqlmap/sqlmap.js

```diff
@@ -57,12 +58,18 @@ function executeMap (command, config, urlDescription, done) {
     params.push(`--data=${urlDescription.data}`)
   }
 
-  console.log(chalk.green('⏳  executing sqlmap with: ', (['' + command].concat(params)).join(' ')))
+  console.log(
+    chalk.green(
+      '⏳  executing sqlmap with: ',
+      ['' + command].concat(params).join(' ')
+    )
+  )
 
   const sql = spawn(command, params)
+  sql.stdin.end()
```

Comment:
this is the change required to fix the hang issue. without this, sqlmap waits and never exits